knowledge-base/how-to-use-reportsource-objects-with-reportbook.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to use ReportSource objects with ReportBook
+description: How to use ReportSource objects with ReportBook.
+type: how-to
+page_title: How to use ReportSource objects with ReportBook
+slug: how-to-use-reportsource-objects-with-reportbook
+res_type: kb
+---
+
+## Description
+ 
+Adding reports to [ReportBook](../designing-reports-general-explanation) shows a warning in **Visual Studio**.  
```

Comment:
Given that, should the be a troubleshooting article with a title - Adding reports to ReportBook shows "The Reports property is now obsolete. Please, use ReportSources property instead." warning?